### ISSUE TITLE
Show distance from players in /whohas

### ISSUE THREAD

**Issue by Obi2** [TotalFreedom/TotalFreedomMod#724]
As whohas is often used to detect players using spammable items, showing the distance the sender is from players with the specified item would be useful. Using /radar alone does not corroborate the location(s) of the item user(s) as the list of players printed out using /whohas isn't ordered.

**Comment by looperXD** [comment]
Agree, this would be handy when it comes to dealing with pot/bow spammings
